libs/node-custom/pnpm-lock.yaml

```diff
@@ -0,0 +1,298 @@
+lockfileVersion: 5.3
```

Comment:
Is there a reason this lib has its own `pnpm-lock.yaml`?

Reply:
It shouldn't. Will delete.